targets/TARGET_Cypress/TARGET_PSOC6/sb-tools/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# To install next python modules, please execute command:
+# python -m pip install -r requirements.txt
+
+cryptography>=2.4.2
```

Comment:
isn't this duplicate of our req file in the root?

Reply:
generating keys has different package requirements than building, testing, etc.

Reply:
> isn't this duplicate of our req file in the root?

Yes. Actually this have arisen after i added our requirements to main mbed-os requirements.txt to integrate auto installation of our required packages on your CI run.
@SeppoTakalo suggested this workaround here **https://github.com/ARMmbed/mbed-os/pull/11018#issuecomment-514971043**

We are okay with strict versioning of packages. Let me know if i should change something in requirements.txt.

Reply:
Lets lock to the minor version at least as we do with other modules (in case for crypto it would be install any 2.4.x, does that sound sensible?

cc @ARMmbed/mbed-os-tools

Reply:
Yes, it does

Reply:
updated versioning in requirements.txt